webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -106,6 +110,162 @@ class ResponsePage extends Component {
+                if (this.state.outcome.status === 'ACCEPTED') {
+                    return <span><span class="badge badge-pill badge-success">{this.state.outcome.status}</span> {this.formatDate(this.state.outcome.timestamp)}</span>
+                } else if (this.state.outcome.status === 'REJECTED') {
+                    return <span><span class="badge badge-pill badge-danger">{this.state.outcome.status}</span> {this.formatDate(this.state.outcome.timestamp)}</span>
+                } else {
+                    return <span><span class="badge badge-pill badge-warning">{this.state.outcome.status}</span> {this.formatDate(this.state.outcome.timestamp)}</span>
+                }
+            };
+
+            if (this.state.event_type === 'CONTINUOUS_JOURNAL' || this.state.event_type === 'JOURNAL') {
+                return <div className='user-details'>
+                    <div className="user-details">
+                        <button
+                            type="button"
+                            class="btn btn-success"
+                            id="accept"
+                            onClick={(e) => this.submitOutcome('ACCEPTED')}>
+                            Accept
+                        </button>
+                    </div>
+                    <div className="user-details">
+                        <button
+                            type="button"
+                            class="btn btn-warning"
+                            id="accept"
+                            onClick={(e) => this.submitOutcome('ACCEPT_W_REVISION')}>
+                            Accept with Minor Revision
+                        </button>
+                    </div>
+                    <div className="user-details">
+                        <button
+                            type="button"
+                            class="btn btn-warning"
+                            id="reject with encouragement"
+                            onClick={(e) => this.submitOutcome('REJECT_W_ENCOURAGEMENT')}>
+                            Reject with Encouragement to Resubmit
+                        </button>
+                    </div>    
+                    <div className="user-details">
+                        <button
+                            type="button"
+                            class="btn btn-danger"
+                            id="reject"
+                            onClick={(e) => this.submitOutcome('REJECTED')}>
+                            Reject
+                        </button>
+                    </div>    
+                </div>
+            }
+            else if (this.state.event_type === 'CALL') {
+                return <div className='user-details'>
+                    <div className="user-details">
+                        <button
+                            type="button"
+                            class="btn btn-success"
+                            id="accept"
+                            onClick={(e) => this.goBack(e)}>
```

Comment:
Why are these all going Back? (Rather leave out the buttons if they're not functional otherwise it will be very confusing for users). Anyway, may as well make them work because the logic is mostly the same as what you wrote for journals :) For calls, accept and reject can do the same as your journal one. For events, reject and waitlist will be the same, but accept will be a bit different, so you can leave that out and just remove the accept button for events.

Reply:
My bad, I put those as placeholders to go back and make them functional! Will fix this, thanks!